rust/common/cookieless/src/manager.rs

```diff
@@ -425,6 +420,8 @@ pub fn get_redis_identifies_key(hash: &[u8], team_id: TeamId) -> String {
 
 #[cfg(test)]
 mod tests {
+    // TODO these tests should really hit a real redis rather than a mocked one
+
```

Comment:
not a blocker for this PR obvi but for a rainy day – flags has a bunch of code that hits real redis if you want some examples of that.